tests/unit/phpunit/modules/EmailTemplates/EmailTemplateFieldsTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use SuiteCRM\Test\SuitePHPUnitFrameworkTestCase;
+
+require_once 'modules/EmailTemplates/templateFields.php';
+
+class EmailTemplateFieldsTest extends SuitePHPUnitFrameworkTestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+
+        global $current_user;
+        get_sugar_config_defaults();
+        $current_user = BeanFactory::newBean('Users');
+    }
+
+    public function testgenerateFieldDefsJS()
+    {
+        // execute the method and verify that it returns expected results
+        $actual = generateFieldDefsJS2();
+
+        $this->assertGreaterThan(0, strlen($actual));
```

Comment:
It seems that most of the unit tests are using `$this->`, maybe apply this across the board to all the unit tests?

Reply:
Agreed, I've actually already done so for develop along with a [ton of other changes](https://github.com/salesagility/SuiteCRM/pull/9170) recently, will need to see what can be backported for 7.10.x/7.11.x (Unfortunately a lot of the 7.12.x testing framework improvements will only be on develop since we are stuck on old versions of PHPUnit/Codeception due to PHP 5.5.9 support).